README.md

```diff
@@ -79,7 +79,7 @@ We have a [wiki](https://github.com/gfx-rs/wgpu/wiki) that serves as a knowledge
 
 ### Angle
 
-[Angle](angleproject.org) is a translation layer from GLES to other backends, developed by Google.
+[Angle](http://angleproject.org) is a translation layer from GLES to other backends, developed by Google.
```

Comment:
No https available :(

Reply:
Thankfully their redirect goes to a chrome sight which is https